bookkeeper-stats-providers/twitter-ostrich-provider/pom.xml

```diff
@@ -32,6 +32,38 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.libdir>${basedir}/lib</project.libdir>
   </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
@eolivelli I think we can do it at the end of the work. This is just first part of #230. We don't have to be blocked on here. Your comments on moving the configuration and version to pluginManagement in the root pom can be the last pull request on addressing #230